package.json

```diff
@@ -7,7 +7,6 @@
     "name": "GitHub",
     "url": "https://github.com/github/docs"
   },
-  "license": "(MIT AND CC-BY-4.0)",
```

Comment:
Everything else here looks good to me, but we'll need this change reverted before we can merge it.